node-deb

```diff
@@ -555,5 +573,6 @@ if [ "$no_md5sums" -eq 0 ]; then
 fi
 
 log_info 'Building Debian package'
-dpkg-deb --build "$deb_dir" > '/dev/null'
+chmod -R u+rwX,g+rX-w,o+rX-w "$deb_dir"
+fakeroot dpkg-deb --build "$deb_dir" > '/dev/null'
```

Comment:
I think `fakeroot` is a good tool here and makes sense, but as a philosophical complaint (maybe), I'm not sure if I want to implement this just because it's another dependency that can't be pulled in easily with `npm`. I know I use `jq` when I could have written a tiny bit of `node` to do it. Anyway. I'm going to ask a few guys at my work about this to get some opinions (also, feel free to give yours).